README.md

```diff
@@ -1 +1,23 @@
 # eslint-formatter-todo
+
+A ESLint formatter that only reports errors when they are not present in a list of "TODOS" while allowing authors to update the list of "TODOS" from a list of errors automatically.
+
+When introducing new linting rules to your project it is possible that you may find many linting errors. If your CI is setup to prevent commits when lint errors happen, your team will have to fix all errors before you can add these new rules to your project leading to a delayed introduction of the rules.
+
+This formatter allows you to introduce new rules immediately without blocking commits by taking a snapshot of errors and transforming them to warnings. Errors not found in this snapshot will continue to reported as errors.
+
+## Usage
+
+When you introduce new linting rules, execute ESLint by setting `UPDATE_TODO=1` and passing this formatter:
+
+```bash
+$ UPDATE_TODO=1 eslint --format eslint-formatter-todo
+```
+
+This command will transform all reported ERRORS to WARNINGS and will generate files inside the `.lint-todo` directory to track them.
```

Comment:
So `-1` is not an actual valid value for ESLint. Valid values are `0` (off), `1` (warning), and `2` (error). Going with `-1` has the potential of getting undefined behavior in the future while it also doesn't let me set it in typescript since `@types` restrict severity to only be those 3 values. I could set it to `0` though, and force the formatter to show them as `todo`, thoughts?

Reply:
We're aware that it's not a valid value for Severity, but that can be fixed through type extension. 

It's not undefined behavior, because that behavior is only relevant to us for the purposes of this functionality. We're not persisting the severity anywhere, and we're simply changing the severity for the rendered output. 

Ultimately, if this feature pans out we'd like to propose it as a feature in `eslint` proper. It's already getting build into `ember-template-lint`.